docs/misc/faq.md

```diff
@@ -101,7 +101,7 @@ with several powerful functionalities:
  where thousands of people have uploaded useful containers: anything from Redis,
```

Comment:
Which link? the one to the image? or the Glossary? These both look like doc links to me.